src/components/trends/smbg/SMBGRangeAvgContainer.js

```diff
@@ -1,22 +1,7 @@
-/*
- * == BSD2 LICENSE ==
- * Copyright (c) 2017, Tidepool Project
- *
- * This program is free software; you can redistribute it and/or modify it under
- * the terms of the associated License, which is identical to the BSD 2-Clause
- * License as published by the Open Source Initiative at opensource.org.
- *
- * This program is distributed in the hope that it will be useful, but WITHOUT
- * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
- * FOR A PARTICULAR PURPOSE. See the License for more details.
- *
- * You should have received a copy of the License along with this program; if
- * not, you can obtain one from Tidepool Project at tidepool.org.
- * == BSD2 LICENSE ==
- */
-
 import _ from 'lodash';
-import React, { PropTypes, PureComponent } from 'react';
+import PropTypes from 'prop-types';
+
```

Comment:
Total nit-pick here.  It seems that for a number of these the codemod added a space between these 2 imports.  Might be nice to do a global find/replace on these to get rid of the space.

Up to you though.  Obviously is completely functional as-is.

Reply:
Can do